src/oracle/connection/transaction.rs

```diff
@@ -100,10 +100,7 @@ impl TransactionManager<OciConnection> for OCITransactionManager {
             };
             Ok(())
         } else {
-            conn.batch_execute(&format!(
-                "COMMIT diesel_savepoint_{}",
-                transaction_depth - 1
-            ))
+            conn.batch_execute("COMMIT")
```

Comment:
What about using `RELEASE diesel_savepoint_{}` here? This should give us a similar semantic like on postgres?

Reply:
I am sorry, what do you mean? Afaik there is no `RELEASE` for Oracle. What do you want to do?

Reply:
[This](https://docs.oracle.com/cd/E17952_01/mysql-5.1-en/savepoint.html) mentions a `RELEASE savepoint` syntax.

Reply:
Thats mysql ;)
c.f.
https://docs.oracle.com/database/121/SQLRF/statements_9.htm#SQLRF016
https://docs.oracle.com/database/121/CNCPT/transact.htm#CNCPT038
https://docs.oracle.com/database/121/CNCPT/sqllangu.htm#CNCPT1733

Reply:
I'll give your idea a thought. We might need to define/discuss some edge cases here and then test those. Since as of now we use for the "base" transaction OCI and for all other nested transactions SQL. But your idea sounds doable.